docs/book/src/images/kubeadm-control-plane-machines-resources.plantuml

```diff
@@ -0,0 +1,50 @@
+@startuml kubeadm-control-plane-machines-resources
+
+skinparam nodesep 80
+skinparam ranksep 80
+skinparam object {
```

Comment:
Wondering if we should use GV instead:
- cluster.x-k8s.io/v1alpha3
- controlplane.cluster.x-k8s.io/v1alpha3
- bootstrap.cluster.x-k8s.io/v1alpha3

Reply:
I changed to use GV instead at d68df645c201bc0dbb6c1ce240c0f4362c4a6aa5.